variables.tf

```diff
@@ -19,3 +19,8 @@ variable "log_level" {
   description = "Log level of the lambda output, one of: debug, info, warning, error, critical"
   type        = string
 }
+variable "localstack_host" {
```

Comment:
If we can remove the reference in `main.tf` to this variable and just let localstack set the env for us, then this wouldn't need to be in the top-level `variables.tf` file. Which would be nice because then it wouldn't be exposed to regular users. The variable would be relevant only to the localstack config, so would only need to be in the `tests/localstack.tf` file. Interestingly, this is actually something we do more generally in tardigrade implementations with other provider-specific config variables...

Reply:
It would be nice, but I got a terraform error when I did that:

```
E       Error: Value for undeclared variable
E       
E       A variable named "localstack_host" was assigned on the command line, but the
E       root module does not declare a variable of that name. To use this value, add
E       a "variable" block to the configuration.
```

Reply:
Hmm, I think that would imply the variable was _not_ declared in `tests/localstack.tf` at that time... It does need to be declared, but because pytest copies/symlinks `tests/localstack.tf` into the working dir before running terraform, it should work if it is declared there (that's how it was working when I tested it).